src/platform/EFR32/BLEManagerImpl.cpp

```diff
@@ -706,14 +706,10 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
         ChipLogDetail(DeviceLayer, "Start BLE advertissement");
     }
 
-#ifndef EFR32MG24
-    // set_random_address causes issues in BLE driver later on with MG24 family.
-    // To be remove when updateing to GSDK4.0
```

Comment:
Typo: "updating"

Reply:
Comment is going to be deleted with the PR merge.